packages/contracts-bedrock/contracts/L2/CrossDomainOwnable2.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Predeploys } from "../libraries/Predeploys.sol";
+import { L2CrossDomainMessenger } from "./L2CrossDomainMessenger.sol";
+import { AddressAliasHelper } from "../vendor/AddressAliasHelper.sol";
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
```

Comment:
Why do we need this second version? `xDomainMessageSender` is not aliased.

Reply:
Good catch, I fixed this issue